File: MPChartExample/src/com/xxmassdeveloper/mpchartexample/notimportant/ContentItem.java
package com.xxmassdeveloper.mpchartexample.notimportant;

/**
 * Created by Philipp Jahoda on 07/12/15.
 */
public class ContentItem {

    final String name;
    final String desc;
    boolean isSection = false;

    public ContentItem(String n) {
        name = n;
        desc = "";
        isSection = true;
    }

    public ContentItem(String n, String d) {
        name = n;
        desc = d;
    }
}

File: MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/notimportant/ContentItem.kt
package com.xxmassdeveloper.mpchartexample.notimportant

class ContentItem<T : DemoBase> {
    @JvmField
    val name: String
    @JvmField
    val desc: String
    @JvmField
    var isSection = false
    @JvmField
    var clazz: Class<T>? = null

    constructor(n: String) {
        name = n
        desc = ""
        isSection = true
    }

    constructor(n: String, d: String, clazzName: Class<T>) {
        name = n
        desc = d
        clazz = clazzName
    }
}
